content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -85,7 +85,8 @@ different Kubernetes components.
 | `CustomCPUCFSQuotaPeriod` | `false` | Alpha | 1.12 | |
 | `CustomResourceDefaulting` | `false` | Alpha| 1.15 | 1.15 |
 | `CustomResourceDefaulting` | `true` | Beta | 1.16 | |
-| `DefaultPodTopologySpread` | `false` | Alpha | 1.19 | |
+| `DefaultPodTopologySpread` | `true` | Alpha | 1.19 | 1.19 |
```

Comment:
Line 88 should be `false` for the alpha state?

Reply:
fixed

Reply:
👍